xform-marc21-to-xml/src/main/java/edu/stanford/MarcXMLValidator.java

```diff
@@ -0,0 +1,97 @@
+package edu.stanford;
+
+import org.apache.commons.cli.*;
+import org.xml.sax.SAXException;
+
+import javax.xml.XMLConstants;
+import javax.xml.transform.Source;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ *
+ */
+public class MarcXMLValidator {
+
+    // Apache Commons-CLI Options
+    // https://commons.apache.org/proper/commons-cli/introduction.html
+    static CommandLine cmd = null;
+    static Options options = setOptions();
+
+    static Options setOptions() {
+        Options opts = new Options();
+        opts.addOption("h", "help", false, "help message");
+        opts.addOption("i", "inputFile", true, "MARC-XML input file");
+        return opts;
+    }
+
+    static void printHelp() {
+        HelpFormatter formatter = new HelpFormatter();
+        formatter.printHelp(MarcXMLValidator.class.getName(), options);
+    }
+
+    static String parseInputFile(String iFile) {
```

Comment:
Ok, i don't know if it's worth the payoff to create a whole separate class for just this one kind of method.